grails-app/controllers/org/pih/warehouse/order/OrderController.groovy

```diff
@@ -1096,6 +1096,66 @@ class OrderController {
     }
 
     def orderSummary = {
+        Order order = Order.get(params.id)
+        def orderItems = order?.orderItems
+        render(template: "orderSummary", model: [
+            isPurchaseOrder: order.isPurchaseOrder,
+            isPutawayOrder: order.isPutawayOrder,
+            orderItems: orderItems,
+            anySupplierCode: orderItems?.any { it.productSupplier?.supplierCode },
+            anyManufacturerName: orderItems?.any { it.productSupplier?.manufacturerName },
+            anyManufacturerCode: orderItems?.any { it.productSupplier?.manufacturerCode },
+            currencyCode: order.currencyCode ?: grailsApplication.config.openboxes.locale.defaultCurrencyCode,
+            subtotal: order.subtotal,
+            totalAdjustments: order.totalAdjustments,
+            total: order.total,
```

Comment:
This could be pushed off until later if we know that this is not a huge performance issue, but ...

... consider looking into some performance improvements for these three "properties" as well. I'm guessing this could at least be combined into a single method that pulls all of the data once and then returns the total, subtotal, and adjustments. 
```
Map getOrderCosts() { 
    // retrieve data necessary to calculate subtotal and adjustments 
    return [
        subtotal: subtotal, 
        adjustments: adjustments,
        total: subtotal + adjustments
    ]
}
```